frontends/ol-search-ui/src/util.tsx

```diff
@@ -144,3 +155,76 @@ export const CertificateIcon = () => (
     src="/static/images/certificate_icon_infinite.png"
   />
 )
+
+export const minPrice = (
+  prices: Array<CoursePrice>,
+  includeDollarSign = false
+) => {
+  if (emptyOrNil(prices)) {
+    return null
+  }
+  const price = Math.min(...prices.map(price => price.price))
+
+  if (price > 0 && price !== Infinity) {
+    return includeDollarSign ? `${formatPrice(price)}` : price
+  } else {
+    return "Free"
+  }
+}
+
+export const getStartDate = (
+  object: LearningResourceResult,
+  objectRun: LearningResourceRun
+): string => {
+  if (object.platform === "ocw") {
+    return `${capitalize(objectRun.semester || "")} ${objectRun.year || ""}`
+  } else if (objectRun.start_date) {
+    return moment(objectRun.start_date).format("MMMM DD, YYYY")
+  } else if (objectRun.best_start_date) {
+    return moment(objectRun.best_start_date).format("MMMM DD, YYYY")
+  }
+  return "Ongoing"
+}
+
+export const getInstructorName = (instructor: CourseInstructor) => {
+  if (instructor.full_name) {
+    return instructor.full_name
+  } else if (instructor.first_name && instructor.last_name) {
+    return `Prof. ${instructor.first_name} ${instructor.last_name}`
```

Comment:
Do we record whether or not the instructor is actually a professor? Some of them are lecturers.....Or graduate students...... 😆 
<img width="490" alt="Screen Shot 2022-08-26 at 1 28 17 PM" src="https://user-images.githubusercontent.com/9010790/186959772-32b0bb42-3a74-4bd9-b35d-5a02712f657a.png">
.

Reply:
(I realize this is how Learn does it right now, so no requested change)